internal/pkg/test/tool/require/require.go

```diff
@@ -322,3 +322,11 @@ func MkfsExt3(t *testing.T) {
 		t.Skipf("mkfs.ext3 is too old and doesn't support -d")
 	}
 }
+
+// FuseOverlayfs check that whether fuse-overlayfs is available.
+func FuseOverlayfs(t *testing.T) {
```

Comment:
This function isn't needed  Use `bin.FindBin("fuse-overlayfs")` from "github.com/apptainer/apptainer/internal/pkg/util/bin"

Reply:
This also make me think that we didn't do anything similar for the fakeroot command.  Maybe there should be UserFakerootCmdProfile and FakerootCmdProfile (that's non-userns) that require that command.  They could probably be included in the PR that adds `--fakeroot` tests for action commands (but then use them also in the build `--fakeroot`.